scripts/install/sabnzbd.sh

```diff
@@ -10,30 +10,45 @@
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
 
-user=$(cut -d: -f1 < /root/.master.info)
-password=$(cut -d: -f2 < /root/.master.info)
-distribution=$(lsb_release -is)
-codename=$(lsb_release -cs)
-#latest=$(curl -s https://sabnzbd.org/downloads | grep -m1 Linux | grep download-link-src | grep -oP "href=\"\K[^\"]+")
-latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | grep -Po '(?<="browser_download_url":).*?[^\\].tar.gz"' | sed 's/"//g')
-latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
-
 . /etc/swizzin/sources/functions/pyenv
 . /etc/swizzin/sources/functions/utils
 
-LIST='par2 p7zip-full python3-dev python3-setuptools python3-pip python3-venv libffi-dev libssl-dev libglib2.0-dev libdbus-1-dev'
+user=$(_get_master_username)
+password=$(_get_user_password ${user})
+#latest=$(curl -s https://sabnzbd.org/downloads | grep -m1 Linux | grep download-link-src | grep -oP "href=\"\K[^\"]+")
+latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | grep -Po '(?<="browser_download_url":).*?[^\\].tar.gz"' | sed 's/"//g')
+latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
+systempy3_ver=$(get_candidate_version python3)
+
+#Version 3.2 is going to raise the min python version to 3.6 so we have to differentiate whether or not to build a pyenv
+if dpkg --compare-versions ${systempy3_ver} lt 3.6.0 && dpkg --compare-versions ${latestversion} ge 3.2.0; then
+    LIST='par2 p7zip-full libffi-dev libssl-dev libglib2.0-dev libdbus-1-dev'
+    PYENV=True
+else
+    LIST='par2 p7zip-full python3-dev python3-setuptools python3-pip python3-venv libffi-dev libssl-dev libglib2.0-dev libdbus-1-dev'
+fi
 
 apt_install $LIST
 
-python3_venv ${user} sabnzbd
+case ${PYENV} in
+    True)
+        pyenv_install
+        pyenv_install_version 3.7.7
+        pyenv_create_venv 3.7.7 /opt/.venv/sabnzbd
+        chown -R ${user}: /opt/.venv/sabnzbd
+        ;;
+    *)
+        python3_venv ${user} sabnzbd
+        ;;
+esac
 
 install_rar
```

Comment:
Can we either get rid of this from the installer or use #555 for it?

Reply:
I don't want a pending PR to block the merging of this script.